packages/ts/generator-typescript-plugin-backbone/src/EndpointMethodRequestBodyProcessor.ts

```diff
@@ -18,54 +18,88 @@ export type EndpointMethodRequestBody = ReadonlyDeep<OpenAPIV3.RequestBodyObject
 export type EndpointMethodRequestBodyProcessingResult = Readonly<{
   parameters: readonly ParameterDeclaration[];
   packedParameters?: ObjectLiteralExpression;
+  initParam: ts.Identifier;
 }>;
 
 export default class EndpointMethodRequestBodyProcessor {
+  readonly #defaultInitParamName = 'init';
```

Comment:
I think we could make it `static` for consistency: I usually follow the logic that since the field cannot be changed during class instantiation, it is unnecessary for it to be a non-static member.